src/transport/IonGasTransport.cpp

```diff
@@ -370,6 +375,85 @@ void IonGasTransport::getMixDiffCoeffs(double* const d)
     }
 }
 
+void IonGasTransport::getMixDiffCoeffsMass(double* const d)
+{
+    update_T();
+    update_C();
```

Comment:
```
                for (size_t j : m_kNeutral) {
                    if (j != k) {
                        sum2 += m_molefracs[j] / m_bdiff(j,k);
                    }
                }
```
It is difficult to use GasTransport::getMixDiffCoeffsMass because of m_kNeutral. Do you think it is worthy to move m_kNeutral to GasTransport?

Reply:
Sorry, you're right. I missed this difference in the implementation, and was only looking at the special handling of the electron species. I think this implementation is fine as is, then -- I don't think we should move `m_kNeutral` up to the GasTransport class.